andela/ah-robotics
# #162949061 Upgrade from Django 1.1 to 2.1

# What does the PR do?
Upgrade the project Django version from 1.1 to 2.1.4

# Description of the Task to be Completed?
- Upgrade Django version
- Change URL resolvers
- Change URL resolvers to use path

# How should this be manually tested?
- Clone the repository 
  ` git clone https://github.com/andela/ah-robotics.git`
- Create a virtual environment in the ah-robotics folder:
   `virtualenv your_virtual_environment`
- Activate the virtual environment created:
  `source your_virtual_environment/bin/activate`
- Install dependencies from the requirements file:
  `pip install -r requirements.txt`
- Switch to `ch-upgrade-django-162949061` branch
  `git checkout ch-upgrade-django-162949061`
- Open the python terminal and type ` python3 -m django --version` to check the version

# Relevant Pivotal Tracker Stories?
 [#162949061](https://www.pivotaltracker.com/n/projects/2235171/stories/162949061)

# Screenshots?
<img width="1039" alt="screen shot 2019-01-10 at 13 07 46" src="https://user-images.githubusercontent.com/14871749/50961500-e8a30500-14d8-11e9-9274-247d25c03e22.png">
